sysbuild/Kconfig.secureboot

```
@@ -339,6 +339,16 @@ config SECURE_BOOT_PUBLIC_KEY_FILES
 	  empty string then only the public key hash corresponding to the private signing key used
 	  to sign the image is included in provision.hex.
 
+config SUPPORT_SECURE_BOOT_BOOTCONF_LOCK_WRITES
+	bool
+	default y if SOC_NRF54L15_CPUAPP
+
+config SECURE_BOOT_BOOTCONF_LOCK_WRITES
+	bool "Disable writes on bootloader's NVM"
```

Having to enable something (setting it to `y`) in order to disable it always twists my brain, especially during late evenings.

Can we re-word while still keeping the same meaning, for example like this:
```suggestion
	bool "Protect bootloader's NVM from writes"
```

---

Fixed